### PR TITLE
Change the link of studies and participants clickable diagrams in the…

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

File: src/components/UserDashboard/charts.js [hunk 1]
```diff
@@ -175,23 +164,41 @@ export const UserInterestsChart = withTheme(({ data, theme }) => {
...
+)(({ data, theme, setSqons, virtualStudy, history }) => {
 
   const onClick = barData => {
     trackBarClick(diagnosesChartCategory, barData);
-    history.push(barData.data.url);
+    addSqon('diagnoses.diagnosis', barData.data.name);
+    history.push('/explore');
+  };
+
+  const addSqon = (field, value) => {
```
Comment (Velythyl) [inline src/components/UserDashboard/charts.js]: Small thing: would extracting the addSqon function and making it top-level in the file a good idea? The addSqon function is also used at line 88, so we could avoid duplicated code this way. I don't know if this is the right move though as addSqon has side effects :)

Comment (nyanofthemoon) [reply src/components/UserDashboard/charts.js]: Would rather not since the components are now connected to the state.

Comment (Velythyl) [reply src/components/UserDashboard/charts.js]: (passing the virtualstudy as a parameter would be required, though)

Comment (Velythyl) [reply src/components/UserDashboard/charts.js]: Alright! LGTM then :)

**Pull request merged**